### ISSUE TITLE
Build error on OSX

### ISSUE THREAD

**Issue by aalmiray** [javafxports/openjdk-jfx#19]
I've encountered an error when attempting to make a full build as per the instructions found in `CONTRIBUTING.md`, that is, running `gradle all test`. The error appears when executing the `test` task on the `web` subproject. If the command is changed to  `gradle all test -x :web:test` then the build succeeds.

	#
	# A fatal error has been detected by the Java Runtime Environment:
	#
	#  SIGSEGV (0xb) at pc=0x000000010e43d93a, pid=79499, tid=33539
	#
	# JRE version: Java(TM) SE Runtime Environment (9.0+181) (build 9+181)
	# Java VM: Java HotSpot(TM) 64-Bit Server VM (9+181, mixed mode, tiered, compressed oops, g1 gc, bsd-amd64)
	# Problematic frame:
	# V  [libjvm.dylib+0x3dd93a]  jni_CallStaticObjectMethodV+0x8d
	#
	# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
	#
	# An error report file with more information is saved as:
	# /Users/aalmiray/dev/github/openjdk-jfx/modules/javafx.web/hs_err_pid79499.log
	#
	# If you would like to submit a bug report, please visit:
	#   http://bugreport.java.com/bugreport/crash.jsp
	#
	
Environment is set to

    ------------------------------------------------------------
    Gradle 4.3
    ------------------------------------------------------------

    Build time:   2017-10-30 15:43:29 UTC
    Revision:     c684c202534c4138b51033b52d871939b8d38d72

    Groovy:       2.4.12
    Ant:          Apache Ant(TM) version 1.9.6 compiled on June 29 2015
    JVM:          9 (Oracle Corporation 9+181)
    OS:           Mac OS X 10.12.5 x86_64

**Comment by rladstaetter** [comment]
I think the same issue was discussed in the openjfx mailing list.

http://mail.openjdk.java.net/pipermail/openjfx-dev/2018-March/021574.html

I could reproduce the bug myself, after building openjfx with webkit the build finished successfully.

**Comment by johanvos** [comment]
That seems to be correct. A follow-up issue might show a nice warning when the web tests are executed without webkit being build.
